src/components/Navbar.js

```diff
@@ -0,0 +1,12 @@
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+// import Home from './Home';
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to remove commented code from your repo, it makes your code messy and hard to read

Reply:
yes sure